### DISCUSSION TITLE
oh-my-posh not working in VS code anymore

### DISCUSSION THREAD

**Discussion by TrifanBogdan24** [JanDeDobbeleer/oh-my-posh#5797] [General]
I use the terminal in all my IDEs, including JetBrains and VS code. Until a week or two ago, everything was alright, but now I encounter issues with rendering the prompt shell in VS Codes's terminal as nice as it is rendered in my computer's terminal.
Perhaps I'm writing the wrong values in VS Code > Settings > Integrated: Font Family.

I don't know...
![broken_in_vs_code](https://github.com/user-attachments/assets/a8054fa4-670f-4abe-af2a-c2a6d090c002)
![working_in_normal_terminal](https://github.com/user-attachments/assets/f7ca5861-a2a3-4507-aeb1-8a8134c61774)

Look at the difference between these two.

I've read a similar issue here: #5274.

Chances are its a VS Code problem.

**Comment by JanDeDobbeleer** [comment]
@TrifanBogdan24 this looks like a font configuration issue. Oh-my-posh can never really break the displaying, that's always the terminal's responsibility.

**Comment by gtusr** [comment]
Any more progress on this issue?  

I'm having the same problem.

**Comment by mayannaoliveira** [comment]
Dear, I fixed the broken characters in terminal after install [Nerd Fonts](https://www.nerdfonts.com/), a easy way to install it is using the terminal command `oh-my-posh font install` and choose your favorite font. The next step in `VS Code > Settings` and change *terminal fonts* to one of Nerd Fonts.
Best regards! 😄👍
